web/src/main/js/store/actions/audit.js

```diff
@@ -87,11 +87,11 @@ export function getAuditData(pageNumber, pageSize, searchTerm, sortField, sortOr
         const errorHandlers = [];
         errorHandlers.push(HTTPErrorUtils.createUnauthorizedHandler(unauthorized));
         errorHandlers.push(HTTPErrorUtils.createForbiddenHandler(() => auditDataFetchError(HTTPErrorUtils.MESSAGES.FORBIDDEN_READ)));
+        const headers = new Headers();
+        headers.append('X-CSRF-TOKEN', csrfToken);
```

Comment:
Since headers is now an object, let's add functions to the object that does this for us. Such as

> headers.addXCrsfToken(csrfToken)